src/test/scala/org/broadinstitute/dsde/firecloud/service/BaseServiceSpec.scala

```diff
@@ -12,9 +12,10 @@ class BaseServiceSpec extends ServiceSpec with BeforeAndAfter {
   val rawlsDao:MockRawlsDAO = new MockRawlsDAO
   val searchDao:MockSearchDAO = new MockSearchDAO
   val thurloeDao:MockThurloeDAO = new MockThurloeDAO
+  val ontologyDao:MockOntologyDAO = new MockOntologyDAO
 
   val app:Application =
-    new Application(agoraDao, rawlsDao, searchDao, thurloeDao, googleServicesDao)
+    new Application(agoraDao, rawlsDao, searchDao, thurloeDao, googleServicesDao, ontologyDao)
```

Comment:
Tests are failing because you changed the signature of `LibraryServiceSupport.indexableDocument`, but didn't change the calling code in the tests.  In `LibraryServiceSpec`, change the class declaration from:
```
import org.scalatest.FreeSpec
...
class LibraryServiceSpec extends FreeSpec
```
to
```
import org.scalatest.FreeSpecLike
...
class LibraryServiceSpec extends BaseServiceSpec with FreeSpecLike 
```
This will bring `ontologyDao` into scope, and you can change the six instances of
```
indexableDocument(w)
```
to
```
indexableDocument(w, ontologyDao)
```

That'll get you past the first error, but then you'll have problems with a missing `UserInfo` ... see my first comment. I'm surprised we don't already have a mock `UserInfo` in our test suite anywhere, but I can't find it (at least right now) ... you can hack this by adding:
```
  implicit val userInfo:UserInfo = UserInfo("fakeToken", "fakeSubjectId")
```
into `BaseServiceSpec`.